pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -16,81 +16,79 @@ let
       symlinkJoin,
       concatTextFile,
       makeSetupHook,
-
+      gradle-unwrapped,
       runCommand,
-    }:
-    this-gradle-unwrapped:
-    lib.makeOverridable (
```

Comment:
Technically when you callPackage something, it becomes overridable. So we didn't need this function returning a function returning an overridable, we can just return the top function.